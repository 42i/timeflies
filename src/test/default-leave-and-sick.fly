day 2012-04-16, sick; Fritz runs
day 2012-04-17, leave; Long walk
day 2012-04-18, off; Ha ha
