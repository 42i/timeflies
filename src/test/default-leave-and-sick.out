default-leave-and-sick.fly:3: ERROR : argument missing in instruction "off".
1 error.
Time at work overview (all):
     when        worked   leave    sick balance
2012-04-16 Mon: ----.-- ----.--    8.00 ----.-- Fritz runs
2012-04-17 Tue: ----.--    8.00 ----.-- ----.-- Long walk
2012-04-18 Wed: ----.-- ----.-- ----.--   -8.00
  week 2012-16: ----.--    8.00    8.00   -8.00
 month 2012-04: ----.--    8.00    8.00   -8.00
         total: ----.--    8.00    8.00   -8.00
     when        worked   leave    sick balance
